java/rocksjni/ratelimiterjni.cc

```diff
@@ -22,3 +23,98 @@ jlong Java_org_rocksdb_GenericRateLimiterConfig_newRateLimiterHandle(
       static_cast<int64_t>(jrefill_period_micros),
       static_cast<int32_t>(jfairness)));
 }
+
+/*
+ * Class:     org_rocksdb_RateLimiter
+ * Method:    newRateLimiterHandle
+ * Signature: (JJI)J
+ */
+jlong Java_org_rocksdb_RateLimiter_newRateLimiterHandle(
+    JNIEnv* env, jclass jclazz, jlong jrate_bytes_per_second,
+    jlong jrefill_period_micros, jint jfairness) {
+  auto* rate_limiter = rocksdb::NewGenericRateLimiter(
+      static_cast<int64_t>(jrate_bytes_per_second),
+      static_cast<int64_t>(jrefill_period_micros),
+      static_cast<int32_t>(jfairness));
+
+  // TODO(AR) `ptr_sptr_rate_limiter` is never deleted :-(
```

Comment:
Maybe improve this comment a little bit?  While the shared_pointer pointed by ptr_sptr_rate_limiter will be deleted, the pointer ptr_sptr_rate_limiter itself is never deleted.